crates/daphne/src/vdaf/mod.rs

```diff
@@ -195,6 +220,15 @@ pub enum VdafPrepState {
     Prio3Field64(Prio3PrepareState<Field64, 16>),
     Prio3Field64HmacSha256Aes128(Prio3PrepareState<Field64, 32>),
     Prio3Field128(Prio3PrepareState<Field128, 16>),
+    Prio3LatestField64(
```

Comment:
Likewise here: the old thing should get the more specific name.

```suggestion
    Prio3Field64(
```